go.mod

```diff
@@ -5,9 +5,11 @@ go 1.22.7
 require (
 	github.com/hashicorp/go-memdb v1.3.4
 	github.com/hashicorp/terraform-json v0.24.0
-	github.com/hashicorp/terraform-plugin-framework v1.13.0
+	github.com/hashicorp/terraform-plugin-framework v1.13.1-0.20250117124025-7e7910eadc9c
```

Comment:
TODO: Update with released framework version, currently pointing to commit from linked PR